src/utils/validations/general-info-step.js

```diff
@@ -0,0 +1,15 @@
+import { nameField } from './common'
+
+export const firstName = (value) => {
+  if (value.trim().length >= 1 && value.trim().length < 3) {
+    return 'common.errorMessages.shortText'
+  }
+  return nameField(value.trim())
+}
+
+export const lastName = (value) => {
+  if (value.trim().length >= 1 && value.trim().length < 3) {
+    return 'common.errorMessages.shortText'
+  }
+  return nameField(value.trim())
+}
```

Comment:
the code for _lastname_ is almost identical to the _firstname_. You can rename the validator to something like `userNameValidator` and reuse it for both fields in form. Also, you can call `trim` once and keep value in an internal variable (to avoid an extra iterations).

```suggestion
export const userNameValidator = (value) => {
   const trimedValue = value?.trim() ?? ''
  if (trimedValue.length >= 1 && trimedValue.length < 3) {
    return 'common.errorMessages.shortText'
  }
  return nameField(trimedValue)
}
```